### DISCUSSION TITLE
啥时候出2.6.14版本

### DISCUSSION THREAD

**Discussion by Ad-closeNN** [Meloong-Git/PCL#3357] [提问] [answered]
啥时候能出2.6.14版本，赞助者还是12月的
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/7bcf5e52-5b20-41f7-99b9-7f3eb81f1ac8)

我估计快出了，~~然后出了就是更多的bug~~

**Comment by allMagicNB** [comment, marked as answer]
这并不是能够确定的数，但是3月1日前一定会出
